src/content/en/ilt/pwa/setting-up-the-labs.md

```diff
@@ -16,112 +16,97 @@ book_path: /web/ilt/pwa/_book.yaml
 
+<aside markdown="1" class="key-point">
+<p>Note: Although not a hard requirement, for general development it can be useful to <a href="tools-for-pwa-developers#disablehttpcache">disable the HTTP cache</a>.</p>
+</aside>
 
 
-Note: Although not a hard requirement, for general development it can be useful to <a href="tools-for-pwa-developers#disablehttpcache">disable the HTTP cache</a></a>.
 
+## Install Git
 
 
 
-## Set up Node & NPM
-
-
-
-
-Install  [Node](https://nodejs.org/en/) and run a local Node server (you may need administrator privileges to do this).
-
-We recommend installing the latest __long term support (LTS)__ version of  [Node](https://nodejs.org/en/) (labelled as "LTS, Recommended for Most Users") rather than the most current version with the latest features (the current version may be unstable).
-
-### Using NVM
-
-If you have an existing version of Node installed that you would like to keep, you can install a Node version manager (for  [macOS and Linux platforms](https://github.com/creationix/nvm) and  [Windows](https://github.com/coreybutler/nvm-windows)). This tool (NVM) lets you install multiple versions of Node, and easily switch between them. If you have issues with a specific version of Node, you can <a href="https://github.com/creationix/nvm#usage">switch to another version</a> with a single command.
-
-Install Node by running one of the following commands from the command line:
-
-If you have installed Node Version Manager (for macOS, Linux, or Windows):
-
-    nvm install node <version>
-
-For example:
-
-    nvm install node 6.11.2
-
-For the Windows version you can specify whether to install the 32-bit or 64-bit binaries.  For example:
-
-    nvm install node 6.11.2 64
-
-### Without NVM
-
-If you did not install NVM, download and install Node from the  [Node.js](https://nodejs.org/en/) website.
-
-This also installs Node's package manager,  [`npm`](https://www.npmjs.com/).
-
-### Check Node and NPM versions
 
-Check that Node and `npm` are both installed by running the following commands from the command line:
+[Install the latest version of Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) on your computer.
 
-    node -v
-    npm -v
 
-If both commands return a version number, then the installations were successful.
+## Clone the repo
 
 
-## Install a simple Node HTTP server
 
 
+Open your computer's command line interface. Clone the course repository with Git using the following command:
 
+    git clone https://github.com/google-developer-training/pwa-training-labs.git
+    cd pwa-training-labs
 
-Install a simple Node server with the following command:
+Some projects in the download contain folders that correspond to checkpoints in the lab (in case you get stuck during the labs, you can refer back to the checkpoints to get back on track).
```

Comment:
The parenthetical is a new sentence. It also feels too important to be in parentheses.